cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       priorityClassName: system-cluster-critical
       containers:
       - name: kubernetes-dashboard
-        image: k8s.gcr.io/kubernetes-dashboard-amd64:v1.8.0
+        image: k8s.gcr.io/kubernetes-dashboard-amd64:v1.8.3
```

Comment:
can you also change the kubernetes-dashboard-certs volume back to a secret (reverting the dashboard change in 
https://github.com/kubernetes/kubernetes/pull/58720/files#diff-7ab88bb7654d97946d6328f11f29d177, now that the dashboard only reads from that mount)